docs/conf.py

```diff
@@ -0,0 +1,102 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, '/Users/yifan/QCRI/neurox/NeuroX/neurox')
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'NeuroX toolkit'
+copyright = '2022, Qatar Computing Research Institute, HBKU'
+author = 'Qatar Computing Research Institute, HBKU'
+release = '1.0.9'
```

Comment:
Should we read this from bump2version somehow to automate for the new version?

Reply:
Apparently I missed this one :) good catch